README.md

```diff
@@ -21,6 +21,24 @@ undirected graphs, and electrical circuits.
 * Create an integration with markdown converters, where the visualized
   models can be used in Latex environments and on static web pages.
 
+### Getting Started
+
+Before you can start visualizing your models inside a web browser, you need to 
+install the following Node packages using NPM (Node Package Manager): http-server
+
+	npm install -g browser-sync
+
+
+### Flow plan:
+
+1. The server listens to changes in the file called **miking-ipm/src/server/model.mc** and will call the parser that transforms the MCore project into an AST and generates a JSON. 
```

Comment:
Why do we hardcode the file names? It would be better if we invoke miking ipm and explicitly give the file name, e.g. 
`mipm model.mc`